### PR TITLE
IBAnimatableApp.xcodeproj -> IBAnimatable.xcodeproj

### PR DESCRIPTION
**Pull request by tbaranes**
This time, everything is working 😬
Also updated travis to make it compile the app instead of the framework. If the app compiles, that means the framework is compiling too.

### CODE REVIEW

**Comment by tbaranes** [comment]
Not sure why travis failed, I run it locally, it's working fine.

File: IBAnimatable.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1228,7 +1228,7 @@
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = JakeLin.IBAnimatable;
+				PRODUCT_BUNDLE_IDENTIFIER = IBAnimatable.IBAnimatable;
```
Comment (JakeLin) [inline IBAnimatable.xcodeproj/project.pbxproj:1231]: I remember I changed that before 😉 Good pick up.

**Review by JakeLin — COMMENTED** [review]
Trivs says "xcodebuild: error: The project named "IBAnimatable" does not contain a scheme named "IBAnimatableApp". The "-list" option can be used to find the names of the schemes in the project.".  Let's have a look how can we fix it.

File: IBAnimatable.xcodeproj/project.pbxproj (hunk 1 above)
Comment (tbaranes) [reply IBAnimatable.xcodeproj/project.pbxproj:1231]: You changed in the framework, that one is in the demo project :)

**Comment by tbaranes** [comment]
I run exactly the same command in my terminal, it was working

**Comment by JakeLin** [comment]
Then let's ship it

**Pull request merged**